seed/challenge-style-guide.md

```diff
@@ -0,0 +1,95 @@
+# A guide to designing freeCodeCamp coding challenges
+
+> “Talk is cheap. Show me the code.” — Linus Torvalds
+
+freeCodeCamp offers 1,200 hours of interactive coding challenges. These are 100% focused on the practical skill of building software. You code the entire time. You learn to code by coding.
+
+You can learn theory through free online university courses. freeCodeCamp will focus instead on helping you learn to code and practice by building apps.
+
+With that practical focus in mind, let’s talk about the requirements for our coding challenges. (Note that these requirements do not apply to our algorithm challenges, checkpoint challenges, or projects.)
+
+## The 2 minute rule
+
+Each challenge should be solvable within 120 seconds by a native English speaker who has completed the challenges leading up to it. This includes the amount of time it takes to read the directions, understand the seeded code, write their own code, and get all the tests to pass.
```

Comment:
@Greenheart - I can re-push my branch, but there's a space between "seconds" and "by" on my local version. Odd.